### DISCUSSION TITLE
Dynamically create DAGS with a DAG.

### DISCUSSION THREAD

**Discussion by pedro-cf** [apache/airflow#39382] [Q&A] [answered]
Is it possible to create persistant dynamic DAGs from a DAG ?

**Comment by andresitorusz** [comment, marked as answer]
Yes, it is possible. You may generate the dynamic DAGs using a dynamic DAG generator by reading the DAG template using YAML, JSON, etc.

Check this blog from Astronomer for more information -> https://docs.astronomer.io/learn/dynamically-generating-dags.
